### PR TITLE
Add quadrophenic

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+			}
+		},
+		secondary: {
+			chance: 20,
+			onHit(target, source) {
+				switch(toID(source.types[0])) {
+					'normal':
+						let type = ["Fighting", "Flying", "Poison", "Ground", "Rock",
+										"Bug", "Ghost", "Steel", "Fire", "Water", "Grass", "Electric",
+										"Psychic", "Ice", "Dragon", "Dark", "Fairy"][this.random(17)];
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						const type = this.sample(Object.values(Dex.data.TypeChart));
```

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+						source.types = [type];
+						this.add('-start', pokemon, 'typechange', type, '[silent]');
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						this.add('-start', pokemon, 'typechange', type, '[silent]');
						break;
```

File: data/mods/ssb/moves.ts [hunk 3]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'fire':
+						target.trySetStatus('brn', source);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						target.trySetStatus('brn', source);
						break;
```

File: data/mods/ssb/moves.ts [hunk 4]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'water':
+						source.addVolatile('aquaring', source);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						source.addVolatile('aquaring', source);
						break;
```

File: data/mods/ssb/moves.ts [hunk 5]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'grass':
+						if (target.hasType('Grass')) return;
+						target.addVolatile('leechseed', source);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						if (target.hasType('Grass')) break;
						target.addVolatile('leechseed', source);
						break;
```

File: data/mods/ssb/moves.ts [hunk 6]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'electric':
+						target.trySetStatus('par', source);
+					'bug':
+						target.side.addSideCondition('stickyweb');
+					'ice':
+						target.trySetStatus('frz', source);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						target.trySetStatus('par', source);
						break;
					'bug':
						target.side.addSideCondition('stickyweb');
						break;
					'ice':
						target.trySetStatus('frz', source);
						break;
```

File: data/mods/ssb/moves.ts [hunk 7]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'poison':
+						target.trySetStatus('tox', source);
+					'dark':
+						target.addVolatile('taunt', source);
+					'ghost':
+						target.trySetStatus('slp', source);
+					'psychic':
+						this.field.setTerrain('psychicterrain');
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						target.trySetStatus('tox', source);
						break;
					'dark':
						target.addVolatile('taunt', source);
						break;
					'ghost':
						target.trySetStatus('slp', source);
						break;
					'psychic':
						this.field.setTerrain('psychicterrain');
						break;
```

File: data/mods/ssb/moves.ts [hunk 8]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'flying':
+						source.side.addSideCondition('tailwind', source);
+					'dragon':
+						target.forceSwitchFlag = true;
+					'steel':
+						target.side.addSideCondition('gmaxsteelsurge');
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						source.side.addSideCondition('tailwind', source);
						break;
					'dragon':
						target.forceSwitchFlag = true;
						break;
					'steel':
						target.side.addSideCondition('gmaxsteelsurge');
						break;
```

File: data/mods/ssb/moves.ts [hunk 9]
```diff
@@ -1112,6 +1112,83 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
+					'rock':
+						target.side.addSideCondition('stealthrock');
+					'ground':
+						target.side.addSideCondition('spikes');
+					'fairy':
+						this.field.setTerrain('mistyterrain');
+					'fighting':
+						source.addVolatile('focusenergy', source);
```
Comment (KrisXV) [inline data/mods/ssb/moves.ts]:
```suggestion
						target.side.addSideCondition('stealthrock');
						break;
					'ground':
						target.side.addSideCondition('spikes');
						break;
					'fairy':
						this.field.setTerrain('mistyterrain');
						break;
					'fighting':
						source.addVolatile('focusenergy', source);
						break;
```

File: data/mods/ssb/random-teams.ts [hunk 10]
```diff
@@ -214,6 +214,14 @@ export const ssbSets: SSBSets = {
 		signatureMove: 'Dungeons & Dragons',
 		evs: {def: 4, spa: 252, spe: 252}, ivs: {atk: 0}, nature: 'Timid',
 	},
+	quadrophenic: {
+		species: 'Porygon', ability: 'Adaptability', item: 'Eviolite', gender: 'N',
+		moves: [
+			'Tri Attack', 'Flamethrower', 'Surf', 'Energy Ball', 'Bug Buzz', 'Aeroblast', 'Thunderbolt', 'Ice Beam', 'Dragon Pulse', 'Power Gem', 'Earth Power', 'Moonblast', 'Dark Pulse', 'Shadow Ball', 'Psychic', 'Aura Sphere', 'Flash Cannon', 'Sludge Bomb',
```
Comment (KrisXV) [inline data/mods/ssb/random-teams.ts]: bruh

**Review by KrisXV — CHANGES_REQUESTED** [review]

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (ghost) [reply data/mods/ssb/moves.ts]: oh thats good

File: data/mods/ssb/random-teams.ts (hunk 10 above)
Comment (ghost) [reply data/mods/ssb/random-teams.ts]: dont judge :<

**Pull request merged**